src/main/java/net/minecraftforge/common/util/math/Size2i.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2019.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.common.util.math;
+
```

Comment:
This is the only class in this package... why is it in it's own package?

Reply:
I wanted to mimic mojang's `net.minecraft.util.math` package which has blockpos and some vector classes. Earlier, I had 2 classes in here but it turned out we only needed 1.

Reply:
I can see your idea, but it's not good it should be in the util package or somewhere else with util classes.

Reply:
Ok I have moved this to the util package so it's not alone in its own package.